modules/perception/onboard/component/BUILD

```diff
@@ -146,7 +146,7 @@ cc_library(
         "//modules/perception/base",
         "//modules/perception/common/sensor_manager",
         "//modules/perception/lib/registerer",
-        "//modules/perception/lib/utils",
+        "//modules/common/util:perf_util",
```

Comment:
ditto. If you made this change with script, maybe just call clang-format for CPP files and buildifier for BUILD files at the end of the script :)

Reply:
> ditto. If you made this change with script, maybe just call clang-format for CPP files and buildifier for BUILD files at the end of the script :)

Forgot to buildify these files in a hurry. :-(